clients/java/signalr/src/main/java/com/microsoft/aspnet/signalr/WebSocketTransport.java

```diff
@@ -24,15 +25,15 @@ public WebSocketTransport(String url, Logger logger) throws URISyntaxException {
         this.logger = logger;
     }
 
-    public URI getUrl(){
+    public URI getUrl() {
         return url;
     }
 
     private URI formatUrl(String url) throws URISyntaxException {
-        if(url.startsWith(HTTPS)){
+        if (url.startsWith(HTTPS)) {
             url = WSS + url.substring(HTTPS.length());
         }
-        else if(url.startsWith(HTTP)){
+        else if (url.startsWith(HTTP)) {
```

Comment:
`else if` and `else` are supposed to be on the same line as `}`

Reply:
I'll apply this to the other pr

Reply:
https://github.com/aspnet/SignalR/pull/2924

Reply:
:/